src/devices/imuBosch_BNO055/imuBosch_BNO055.cpp

```diff
@@ -619,14 +619,14 @@ void BoschIMU::run()
 
 
         // Fill in Gyro values
```

Comment:
It would be great to have a link to the PDF datasheet with relative page number to document where that "16.0" value comes from.